src/main/java/org/hydev/mcpm/client/installer/InstallInteractor.java

```diff
@@ -118,6 +145,13 @@ public InstallResult installPlugin(InstallInput installInput)
      */
     public static void main(String[] args) {
         new File(FILEPATH).mkdirs();
+        Consumer<String> log = new Consumer<>() {
+            @Override
+            public void accept(String s) {
+                System.out.println(s);
+            }
+        };
```

Comment:
IntelliJ might offer a simpler solution too!

```suggestion
        Consumer<String> log = s -> System.out.println(s);
```